topi/tests/python/test_topi_depthwise_conv2d.py

```diff
@@ -122,7 +122,6 @@ def depthwise_conv2d_with_workload_nhwc(batch, in_channel, in_height, channel_mu
     # placeholder
     Input = tvm.placeholder((batch, in_height, in_width, in_channel), name='Input')
     Filter = tvm.placeholder((filter_height, filter_width,filter_channel, channel_multiplier), name='Filter')
-    DilatedFilter = topi.nn.dilate(Filter, (1, 1, dilation, dilation), name='DilatedFilter')
```

Comment:
@merrymercy previous dilation is incorrect, I have updated to [dilation, dilation, 1, 1] but current fallback schedule generated invalid ptx on cuda

Reply:
The filter is incorrect. The dilation is correct.

Reply:
nchw and nhwc have different filter layouts, the filter here is consistent with nn.depthwise_conv2d_nhwc.
https://github.com/dmlc/tvm/blob/b840e9602e357c50124d0c7fb131c52321062570/topi/python/topi/nn/depthwise_conv2d.py#L80